src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -127,34 +127,26 @@
                             (ds/rename-columns {:popn-migrated :popn}))]
     {:latest-year-popn popn-w-migrants}))
 
-(defworkflowfn join-popn-latest-year
+(defworkflowfn join-by-year
   "Takes in a dataset of population for previous years and a dataset of
    projected population for the next year of projection.
    Returns a datasets that appends the second dataset to the first one."
   {:witan/name :ccm-core/join-years
    :witan/version "1.0.0"
    :witan/input-schema {:latest-year-popn PopulationSchema
-                        :population PopulationSchema}
-   :witan/output-schema {:population PopulationSchema}
-   :witan/exported? true}
-  [{:keys [latest-year-popn population]} _]
-  {:population (ds/join-rows population latest-year-popn)})
-
-(defworkflowfn join-births-year-1-0-0
-  "Takes in a dataset of births for previous years and a dataset of
-   projected births for the next year of projection.
-   Returns a datasets that appends the second dataset to the first one."
-  {:witan/name :ccm-core/join-years-births
-   :witan/version "1.0.0"
-   :witan/input-schema {:births BirthsBySexSchema
+                        :population PopulationSchema
+                        :births BirthsBySexSchema
                         :historic-births BirthsSchema
                         :loop-year (s/constrained s/Int m-utils/year?)}
-   :witan/output-schema {:historic-births BirthsSchema}}
-  [{:keys [births historic-births loop-year]} _]
+   :witan/output-schema {:population PopulationSchema
+                         :historic-births BirthsSchema}
+   :witan/exported? true}
```

Comment:
I wasn't sure if it made more sense just to get ride of them all at the same time?